tests/format/js/multiparser-markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -109,7 +109,7 @@ markdown\`
 
 =====================================output=====================================
 markdown\`
-  const cssString = css\\\` background-color: \\$\\{color('base')\\}\\\`;
+  const cssString = css\\\`  background-color: \\$\\{color('base')\\}\\\`;
```

Comment:
Shouldn't this call `markdown -> js -> css` and remove the leading space? Because it's not valid css?

Reply:
This is a crazy test. `const cssString = ...` is Markdown here, not JS. The Markdown-in-JS embedder also includes logic for ignoring indentation (no idea what's the story behind that). That's why the output is like that.

Reply:
Ah, you explained before... forgot. Sorry.